cms/djangoapps/contentstore/views/certificates.py

```diff
@@ -67,11 +68,17 @@ def _delete_asset(course_key, asset_key_string):
     remove asset by calling delete_asset method of assets module.
     """
     if asset_key_string:
-        # remove first slash in asset path
-        # otherwise it generates InvalidKeyError in case of split modulestore
-        if '/' == asset_key_string[0]:
-            asset_key_string = asset_key_string[1:]
-        asset_key = AssetKey.from_string(asset_key_string)
+        try:
+            asset_key = AssetKey.from_string(asset_key_string)
+        except InvalidKeyError:
```

Comment:
codecov is telling me that this exception branch is never hit by unit tests.
Would it make sense to add, or is that going to be too difficult?